app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -847,9 +847,9 @@ def build_toolbar_hide_button(id)
       when "vm_reconfigure"
         return true unless @record.reconfigurable?
       when "vm_retire"
-        return true unless @record.is_available?(:retire)
+        return true unless @record.supports_retire?
       when "vm_retire_now"
-        return true unless @record.is_available?(:retire_now)
+        return true unless @record.supports_retire?
```

Comment:
Good question and a confusion about this old validate methods. the now version would check for this very instance whereas the other one could also return true if it's possible but not right now.